labelbox/schema/project.py

```diff
@@ -400,16 +402,27 @@ def _validate_datetime(string_date: str) -> bool:
                          self.uid)
             time.sleep(sleep_time)
 
-    """
-    Creates a project run export task with the given params and returns the task.
-    
-    >>>    export_task = export_v2("my_export_task", filter={"media_attributes": True})
-    
-    """
-
     def export_v2(self,
                   task_name: Optional[str] = None,
+                  filters: Optional[ProjectExportFilters] = None,
```

Comment:
is the intend of default `= None` to allow to pass filters value as `None` as opposed to just missing variable `filters`?  If not, a default is not required. Also see here https://docs.python.org/3.9/library/typing.html#typing.Optional 
PS This seems to be the pattern we have been following, so just curious

Reply:
If the default value is not provided, a function call fails with:
```
 export_v2() missing 2 required positional arguments: 'task_name' and 'params'
```